ledctl documentation improvements

Remove invalid option from the manpage. Show short options in usage().

Author: Jan Synacek <jsynacek@redhat.com>

--- ledmon-0.77/doc/ledctl.pod	2013-02-12 14:45:22.000000000 +0100
+++ ledmon-0.77/doc/ledctl.pod.new	2013-04-19 10:25:41.784551587 +0200
@@ -258,11 +258,6 @@
 Sets a path to local log file. If this option is specified the global log
 file F</var/log/ledctl.log> is not used.
 
-=item B<--quiet>
-
-Turns off all messages sent to C<stdout> or C<stderr> out. The messages
-will be still logged to local file and syslog facility.
-
 =item B<-h> or B<--help>
 
 Prints this text out and exits.
--- ledmon-0.77/src/ledctl.c	2013-02-12 14:45:28.000000000 +0100
+++ ledmon-0.77/src/ledctl.c.new	2013-04-19 10:28:07.895707605 +0200
@@ -179,13 +179,13 @@
 	printf("Mandatory arguments for long options are mandatory for" \
 	       " short options, too.\n\n");
 	printf
-	    ("--log=PATH\t\t\t  Use local log file instead\n" \
+	    ("--log=PATH,  -l PATH\t\t  Use local log file instead\n" \
 	     "\t\t\t\t  /var/log/ledctl.log global file.\n");
 	printf("--config=PATH,  -c PATH\t\t  Use alternate configuration" \
 	       " file (not yet\n\t\t\t\t  implemented).\n");
-	printf("--help\t\t\t\t  Displays this help text.\n");
+	printf("--help,  -h\t\t\t  Displays this help text.\n");
 	printf
-	    ("--version\t\t\t  Displays version and license information.\n\n");
+	    ("--version,  -v\t\t\t  Displays version and license information.\n\n");
 	printf("Patterns:\n"
 	       "\tCommon patterns are:\n"
 	       "\t\tlocate, locate_off, normal, off, degraded, rebuild,\n" ""
